pkgs/by-name/gp/gpgme2/package.nix

```
@@ -0,0 +1,116 @@
+{
+  lib,
+  stdenv,
+  fetchurl,
+  autoreconfHook,
+  libgpg-error,
+  gnupg,
+  pkg-config,
+  glib,
+  pth,
+  libassuan,
+  which,
+  texinfo,
+  buildPackages,
+  qtbase ? null,
```

```suggestion
  qtbase,
```
When should that be null?